app/apollo/resolvers/channel.js

```diff
@@ -15,7 +15,7 @@
  */
 
 const _ = require('lodash');
-const uuid = require('uuid').v4;
+const UUID = require('uuid').v4;
```

Comment:
Needs to be updated to this format `const { v4: uuid } = require('uuid');`. The current format is deprecated and throws warnings now